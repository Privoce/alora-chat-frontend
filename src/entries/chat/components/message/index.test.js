import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import {
	MessageComponent,
	MessageSketch
} from 'entries/chat/components';

describe('<MessageComponent />', () => {
	const modelI = {
		right: true,
		text: 'Hey there! how are you you you you?',
		time: '12:23'
	};

	const getWrapper = (model) => {
		return shallow((
			<div>
				<MessageComponent {...model} />
				<MessageSketch {...model} />
			</div>
		));
	};

	it('renders correctly', () => {
		const wrapper = getWrapper(modelI);
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
