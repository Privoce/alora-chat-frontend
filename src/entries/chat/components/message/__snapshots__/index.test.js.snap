// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MessageComponent /> renders correctly 1`] = `
<div>
  <MessageComponent
    right={true}
    text="Hey there! how are you you you you?"
    time="12:23"
  />
  <MessageSketch
    right={true}
    text="Hey there! how are you you you you?"
    time="12:23"
  />
</div>
`;
